docs/Simulation/Software-RTL-Simulation.rst

```diff
@@ -0,0 +1,134 @@
+.. _sw-rtl-sim-intro:
```

Comment:
Was this just moved or did it change?

Reply:
I essentially merge the "Running-A-Simulation" file into this one (and we're eliminating the "Running-A-Simulation" file)